### DISCUSSION TITLE
How to handle gap in the time series data when building MMM model

### DISCUSSION THREAD

**Discussion by yongqiangzhangzazzle** [google/meridian#213] [Q&A] [answered]
When building the MMM model based on the spend and sale data, we found the spend and sales behave quite differently during November and December. So one idea is to only build MMM model without November and December's data. However, the ads carryover effect can't be captured due to the gap. Any suggestions on how to proceed this?

**Comment by mikewurm** [comment, marked as answer]
You can set use the `holdout_id` argument of spec.ModelSpec to treat the entire months of November and December as holdout observations.  This will ensure that the *sales* of those months has no effect on the model fit, but the *media* in November/December will still be considered in the adstock for January and the following months.

If you don't want the November/December media to be counted in the following months' adstock, you can set the November/December media to all zeros.

**Reply by yongqiangzhangzazzle** [reply]
Thanks @mikewurm This works for me.
